src/utils/helpers.test.js

```diff
@@ -56,3 +69,123 @@ describe('getUKFormattedEuros', () => {
     expect(result).toBe('€123.45');
   });
 });
+
+describe('getCurrentMonth', () => {
+  afterEach(() => {
+    global.Date.now = realDateNow;
+  });
+
+  test('if the current date is 2022-06-01, it should return "June 2022"', () => {
+    const date = '2022-06-01';
+    global.Date.now = getDateNowStub(date);
+
+    const expected = 'June 2022';
+    const actual = getCurrentMonth();
+
+    expect(actual).toBe(expected);
+  });
+  test('if the current date is 2024-02-29, it should return "February 2024"', () => {
```

Comment:
there is lots of empty lines in the tests (I see arrange, act, etc) but there is no space between the tests. this seems a little like a book which has lots of spaces within a chapter, but the next chapter starts right at the end of the last sentence of the previous chapter.

Reply:
Gotcha, I'll space out the test blocks.